ftw/usermigration/browser/migration.py

```diff
@@ -93,6 +110,26 @@ def __init__(self, context, request):
         self.results_users = {}
         self.results_properties = {}
 
+    def _get_mapping_from_form(self, formdata):
```

Comment:
maybe rename this to `_get_manual_mapping` or `_get_user_input_mapping` or `...` -  `from_form` does not read too well.